ubuntu/entry.sh

```diff
@@ -6,7 +6,7 @@ echo "DYNAMODB_TABLE=$DYNAMODB_TABLE"
 echo "AWS_REGION=$REGION"
 
 function get_region {
-    export AZ=$(curl http://169.254.169.254/latest/meta-data/placement/availability-zone 2>/dev/null)
+    export AZ=$(wget -O - http://169.254.169.254/latest/meta-data/placement/availability-zone 2>/dev/null)
```

Comment:
I saw a similar issue with swarm leader migration so I fixed the issue there using option 2.

Reply:
If you are using option 2 then update to the following block in Dockerfile
```
# Note: Install libltdl-dev so that docker binary from host can base used in the image
RUN export DEBIAN_FRONTEND=noninteractive && \
    apt-get update && \
    apt-get install -y jq libltdl-dev python3-pip wget curl && \
    pip3 install -U pip && \
    pip install awscli
```

Reply:
Any update on this or should I fix the bug?

Reply:
Feel free to do it :-)

Reply:
Thanks closing this review.